dialect.go

```diff
@@ -63,7 +63,7 @@ func (pg PostgresDialect) insertVersionSQL() string {
 }
 
 func (pg PostgresDialect) dbVersionQuery(db *sql.DB) (*sql.Rows, error) {
-	rows, err := db.Query(fmt.Sprintf("SELECT version_id, is_applied from %s ORDER BY id DESC", TableName()))
```

Comment:
why these changes?

Reply:
With up-all we able to execute an earlier migration after the last one migration, so in db we can get something like this
id = 1; version_id = 20190325235959
id = 2; version_id = 20190324235959
Originally this sql returned list of migrations from newest to oldest sorted by id field. It worked because we were not able to execute earlier migration after the last one.
To keep it that way for example above we need to sort migrations not by id, but by version_id.
But table can contain record about the same version several times because of non-transaction migrations. So we can have table like this
id = 1; version_id = 20190325235959; is_applied = true;
id = 2; version_id = 20190324235959; is_applied = true;
id = 3; version_id = 20190325235959; is_applied = false;
So to keep all this condition work, we need to add sort by id field.